diff --git a/node_modules/@bcherny/json-schema-ref-parser/lib/index.d.ts b/node_modules/@bcherny/json-schema-ref-parser/lib/index.d.ts
index a62a8fd..1f98f80 100644
--- a/node_modules/@bcherny/json-schema-ref-parser/lib/index.d.ts
+++ b/node_modules/@bcherny/json-schema-ref-parser/lib/index.d.ts
@@ -1,6 +1,6 @@
 import { JSONSchema4, JSONSchema4Object, JSONSchema4Type, JSONSchema6, JSONSchema6Object, JSONSchema6Type, JSONSchema7, JSONSchema7Object, JSONSchema7Type } from "json-schema";
 
-export = $RefParser;
+export default $RefParser;
 
 /**
  * This is the default export of JSON Schema $Ref Parser. You can creates instances of this class using new $RefParser(), or you can just call its static methods.
